Source/Core/Core/PowerPC/Jit64/Jit_LoadStorePaired.cpp

```diff
@@ -125,25 +129,68 @@ void Jit64::psq_stXX(UGeckoInstruction inst)
   // In memcheck mode, don't update the address until the exception check
   if (update && !jo.memcheck)
     MOV(32, gpr.R(a), R(RSCRATCH_EXTRA));
-  // Some games (e.g. Dirt 2) incorrectly set the unused bits which breaks the lookup table code.
-  // Hence, we need to mask out the unused bits. The layout of the GQR register is
-  // UU[SCALE]UUUUU[TYPE] where SCALE is 6 bits and TYPE is 3 bits, so we have to AND with
-  // 0b0011111100000111, or 0x3F07.
-  MOV(32, R(RSCRATCH2), Imm32(0x3F07));
-  AND(32, R(RSCRATCH2), PPCSTATE(spr[SPR_GQR0 + i]));
-  MOVZX(32, 8, RSCRATCH, R(RSCRATCH2));
 
-  if (w)
+  if (gqrIsConstant)
   {
-    // One value
-    CVTSD2SS(XMM0, fpr.R(s));
-    CALLptr(MScaled(RSCRATCH, SCALE_8, (u32)(u64)asm_routines.singleStoreQuantized));
+// Paired stores don't yield any real change in performance right now, but if we can
+// improve fastmem support this might change
```

Comment:
Really?  I would have expected some sort of difference, especially with MMU enabled.

A couple potential issues blocking performance improvements:
- you don't actually get fastmem with SAFE_LOADSTORE_NO_SWAP at the moment (but I think you have a fix for that?)
- It might be beneficial to use WriteToConstAddress in some situations.